content/docs/learning-guide/tutorials/assets/actor-assets.md

```diff
@@ -0,0 +1,92 @@
+---
+linkTitle: Actor Processing
+title: Customize Actor Asset Processing
+description: Learn to customize how actor assets are processed for Open 3D Engine (O3DE) with Scene Settings.
+weight: 300
+toc: true
+---
+
+This tutorial demonstrates how to use [**Scene Settings**](/docs/user-guide/assets/scene-settings/scene-settings) to customize actor asset processing for **Open 3D Engine (O3DE)**. An actor contains one or more meshes bound to a skeleton. You can use any actor asset for this tutorial. The primary 3D scene format supported by O3DE is `.fbx`, so it's recommended to use an actor asset saved to a `.fbx` file. If you don't have your own actor source asset, you can use a character asset from a provider such as [Mixamo](https://www.mixamo.com/#/?page=1&type=Character).
+
+{{< note >}}
+Understanding the [best practices](/docs/user-guide/assets/scene-settings/source-asset-best-practices#actors) for creating actor source assets can mitigate issues you might encounter when processing actors for O3DE. For technical details about the data supported by actors, refer to the [supported 3D scene data](/docs/user-guide/assets/scene-settings/scene-format-support#supported-3d-scene-data) table.
+{{< /note >}}
+
+| O3DE Experience | Time to Complete | Feature Focus | Last Updated |
+| - | - | - | - |
+| Beginner | 10 Minutes | Customized processing of actor assets from `.fbx` files with Scene Settings. | January 4, 2023 |
+
+## Preparation
+
+Place your actor `.fbx` source asset in a [scan directory](/docs/user-guide/assets/pipeline/scan-directories) for your project, such as the `Assets` subdirectory, so it can be detected by [**Asset Processor**](/docs/user-guide/assets/asset-processor).
+
+## Processing an actor asset
+
+When you placed your actor `.fbx` source asset in your project's `Assets` directory, Asset Processor detected the asset, examined its contents, and processed it with a default set of rules. To customize the rules that process the actor asset, do the following:
```

Comment:
Is there something you could say - in this paragraph and/or in the intro - about why, how often, or under what conditions you might want to change the actor asset processing rules to use settings that are not the default set of rules? Is this common? When?

Reply:
Additional context added.